src/http/server.cr

```diff
@@ -186,6 +182,43 @@ class HTTP::Server
     server.local_address
   end
 
+  {% unless flag?(:without_openssl) %}
+    # Creates a `OpenSSL::SSL::Server` and adds it as a socket.
+    #
+    # The SSL server wraps a `TCPServer` listenting on `host:port`.
+    #
+    # ```
+    # server = HTTP::Server.new { }
+    # context = OpenSSL::SSL::Context::Server.new
+    # context.certificate_chain = "openssl.crt"
+    # context.private_key = "openssl.key"
+    # server.bind_ssl "127.0.0.1", 8080, context
+    # ```
+    def bind_ssl(host : String, port : Int32, context : OpenSSL::SSL::Context::Server = OpenSSL::SSL::Context::Server.new, reuse_port : Bool = false) : Socket::IPAddress
```

Comment:
Can the context be accessed and changed after `bind_ssl` is called? 
Or does the default `OpenSSL::SSL::Context::Server.new` load some system information? 

Otherwise I don't see why a default `OpenSSL::SSL::Context::Server` is defined.

Reply:
Yeah, it probably doesn't make much sense. The context can't be accessed afterwards.